include/xtensor-sparse/xlil.hpp

```diff
@@ -25,13 +27,29 @@ namespace xt
         using difference_type = std::ptrdiff_t;
 
         using shape_type = svector<size_type>;
+        using strides_type = svector<size_type>;
+        using backstrides_type = svector<size_type>;
```

Comment:
I don't think it makes sense to have strides and backstrides in sparse tensors.

Reply:
I let `strides` and `backstrides` to compute the new keys easily when we reshape. I will see if we can remove them.